assets/schema_input.json

```
@@ -4,137 +4,153 @@
-                        "errorMessage": "Sample name must be provided and cannot contain spaces"
-                    },
-                    "latin_name": {
-                        "type": "string",
-                        "errorMessage": "The scientific name for the assembly"
-                    },
-                    "classT": {
-                        "type": "string",
-                        "errorMessage": "The Clade of the assembly. Used as the syntenic group and to complete the gene_alignment data dir."
-                    },
-                    "TicketType": {
-                        "type": "string",
-                        "errorMessage": "Not currently in use. Single word description of associated project."
-                    }
+    "properties": {
+        "assembly": {
+            "type": "object",
+            "properties": {
+                "assem_level": {
+                    "type": "string",
+                    "errorMessage": "Assembly level must be provided"
+                },
+                "assem_version": {
+                    "type": "integer",
+                    "errorMessage": "Assembly version must be provided"
+                },
+                "sample_id": {
+                    "type": "string",
+                    "errorMessage": "Sample name must be provided and cannot contain spaces"
+                },
+                "latin_name": {
+                    "type": "string",
+                    "errorMessage": "The scientific name for the assembly"
+                },
+                "defined_class": {
+                    "type": "string",
+                    "errorMessage": "The Clade of the assembly. Used as the syntenic group and to complete the gene_alignment data dir."
+                },
+                "project_id": {
+                    "type": "string",
+                    "errorMessage": "Project name must be provided "
                 }
-            },
-            "reference_file": {
-                "type": "string",
-                "pattern": "^\\S+\\.f(ast)a$",
-                "errorMessage": "Assembly input file, decompressed"
-            },
-            "assem_reads": {
-                "type": "object",
-                "properties": {
-                    "pacbio": {
-                        "type": "string",
-                        "errorMessage": "Path to folder containing fasta.gz files"
-                    },
-                    "hic": {
-                        "type": "string",
-                        "errorMessage": "Path to folder containing .cram and .crai files"
-                    },
-                    "supplementary": {
-                        "type": "string",
-                        "errorMessage": "Not currently in use. Placeholder for future use"
-                    }
+            }
+        },
+        "reference_file": {
+            "type": "string",
+            "pattern": "^\\S+\\.f(ast)a$",
```

If you haven't already, try this?
```suggestion
            "pattern": "^\\S+\\.(fa|fasta|fna)$",
```